sdk/communication/azure-communication-messages/tests/test_messages_client_async.py

```diff
@@ -156,7 +156,7 @@ async def test_image_send_message_async(self):
     @recorded_by_proxy_async 
     async def test_download_media_async(self):
         phone_number: str = "+14254360097"
-        input_media_id: str = "315a0b04-d80d-4573-bcf1-1ce33d3f19f0"
+        input_media_id: str = "8f8c29b2-c2e4-4340-bb28-3009c8a57283"
```

Comment:
Is this secret that we need to sanitize?

Reply:
Hi @xiangyan99 , This is GUID that uniquely identifies media message image and expiry in 30 days. So this is not a secret.

Reply:
(I believe you already did that :)) please double check that they are already invalid.

Reply:
"RequestUri": "https://sanitized.unitedstates.communication.azure.com/messages/streams/sanitized?api-version=2024-02-01",
Added it.